main.py

```diff
@@ -297,17 +297,19 @@ def eval_worker(gpu, ngpu_per_node, args, res_queue=None):
 
 
 def writer(args, res_queue):
-    result_path = os.path.join(args.exp_dir, args.result_fname)
-    myprint("Writer process standing by... %s" % result_path, color='yellow')
 
     # We need dimensions of the inference dataset.
     if args.infer:
         files_path = args.infer_files
     else:
         assert False, "writer called but infer = False. Not sure what file to write?"
+
     total_size = 0
     write_dimension = 0
     for fname in files_path:
```

Comment:
Since the writer is now for just one file at a time, I don't think we need this loop. It would only be writing the output for one of the input files, so that's all the info it needs

Reply:
correct, done.